build/Common.prod.props

```diff
@@ -2,7 +2,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <ItemGroup>
-    <PackageReference Include="MinVer" Version="2.0.0">
+    <PackageReference Include="MinVer" Version="2.3.0">
```

Comment:
was this the rootcause of the issue?

Reply:
i saw that just updating it, it started to print the correct values. Still checking if i have to configure more stuff

Reply:
I saw that the assemblyVersion is still missing, so, it's not complete yet.

Reply:
the reason that this wasn't working correctly: https://github.com/adamralph/minver/issues/342, looks like for netcore3.1, the way it builds make it to don't run properly, so the new version started to work correctly.

Reply:
@cijothomas , with the new commit it solves the issue with the version 0.0.0.0